docs/release-notes/release-notes-1.0.0.md

```diff
@@ -0,0 +1,69 @@
+# DCRDEX v1.0.0
+
+This release introduces a major version upgrade. Backwards compatibility is
+not strictly preserved.
+
+There are many new features, improvements, and bug fixes in since v0.6.
+
+## Features and Improvements
+
+- Add Decred staking
+- Add Decred mixing
+- Add Polygon and Polygon tokens USDC, USDT, WETH, WBTC
+- Add Ethereum token USDT
+- Implement multi-balance contracts for Ethereum and Polygon to reduce provider request rate
+- Use free RPC providers by default for Ethereum and Polygon to enable one-click wallet creation
+- Implement Zcash shielded pool features and refactor wallet to be shielded-by-default
+- Add Digibyte, Firo, Zclassic wallets
+- Expose wallet tx history
```

Comment:
Maybe transaction history.  I generally think it's get practice to avoid too many dev code abbreviations in release notes (unless they're acronyms that are previously defined).